pkg/volume/nfs/nfs.go

```diff
@@ -122,9 +124,12 @@ func (nfsVolume *nfs) SetUpAt(dir string) error {
 	if mountpoint {
 		return nil
 	}
-	exportDir := nfsVolume.exportPath
 	os.MkdirAll(dir, 0750)
-	err = nfsVolume.mounter.Mount(nfsVolume.server, exportDir, dir, nfsVolume.readOnly)
+	flags := uintptr(0)
+	if nfsVolume.readOnly {
+		flags = mount.FlagReadOnly
+	}
+	err = nfsVolume.mounter.Mount(fmt.Sprintf("%s:%s", nfsVolume.server, nfsVolume.exportPath), dir, "nfs", flags, "")
```

Comment:
IIUC, the mounter interface is an interface for different platform.  Different fs type or plugin should implement volume builder/cleaner interface.  Implementation-wise, it's definitely not wrong if NFS using its own mounter; but I think it breaks the structure.

I reverted the changes to nfs_mount, I'll create another PR.   Thanks for review!